ci/scripts/connect-release-init-v9.js

```diff
@@ -4,10 +4,17 @@ const child_process = require('child_process');
 const path = require('path');
 const fs = require('fs');
 
-const { exec } = require('./helpers');
+const { exec, triggerWorkflowSync } = require('./helpers');
 
 const ROOT = path.join(__dirname, '..', '..');
 
+const workflowsOnReleaseBranch = [
+    '.github/workflows/connect-dev-release-test.yml',
+    '.github/workflows/connect-test.yml',
+    '.github/workflows/connect-transport-e2e-test.yml',
+    '.github/workflows/connect-web-e2e-test.yml',
+];
+
```

Comment:
A next step not to have this here could be to create a new action that will trigger all the connect test workflows so we could run only one workflow from here.